test/cli/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+cmake_minimum_required (VERSION 3.8)
+
+add_cli_test (index_test.cpp)
+target_use_datasources (index_test FILES db_nucl.fasta.gz)
+target_use_datasources (index_test FILES db_nucl_bs.fasta.gz)
+target_use_datasources (index_test FILES db_prot.fasta.gz)
+target_use_datasources (index_test FILES db_nucl_bs_bifm.fasta.gz.lba)
+target_use_datasources (index_test FILES db_nucl_bs_fm.fasta.gz.lba)
+target_use_datasources (index_test FILES db_nucl_bifm.fasta.gz.lba)
+target_use_datasources (index_test FILES db_nucl_fm.fasta.gz.lba)
+target_use_datasources (index_test FILES db_prot_bifm.fasta.gz.lba)
+target_use_datasources (index_test FILES db_prot_fm.fasta.gz.lba)
+target_use_datasources (index_test FILES db_trans_bifm.fasta.gz.lba)
+target_use_datasources (index_test FILES db_trans_fm.fasta.gz.lba)
```

Comment:
Maybe put multiple files on one line? Or use linebreaks and write filenames on individual line (without repeating `target_use...`).

Reply:
:nail_care: